Adds Rubik font v2.200

kuzzmi opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change

I am a huge fan of Rubik font and would like to add it to nixpkgs for everybody to use.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Tested on a fresh NixOS installation both through Home Manager and from NixOS main configuration file
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

kuzzmi committed:
rubik-font: init at 2.200

SuperSandro2000 commented on pkgs/data/fonts/rubik/rubik.nix:
```diff
@@ -0,0 +1,107 @@
+with import <nixpkgs> {};
```
```suggestion
{ stdenv, ... }:
```

SuperSandro2000 commented on pkgs/data/fonts/rubik/rubik.nix:
```diff
@@ -0,0 +1,107 @@
+    cp -a *.ttf $out/share/fonts/truetype/
+  '';
+
+  outputHashAlgo = "sha256";
+  outputHashMode = "recursive";
+  outputHash = "0hd7969zdwzsyp5dsjlxhpg23byk1i05n2bb42sfr793pvc5g20d";
+
```
```suggestion
```

SuperSandro2000 commented on pkgs/data/fonts/rubik/rubik.nix:
```diff
@@ -0,0 +1,107 @@
+
+  meta = with lib; {
+    homepage = "https://fonts.google.com/specimen/Rubik";
+    description = "Rubik Fonts";
```
```suggestion
    description = "Rubik Font - is a 5 weight Roman + Italic family.";
```

SuperSandro2000 reviewed (changes requested):
Please follow the contributing guide:
- name the commit ``rubik-font: init at 2.200``
- add an entry to top-level.nix
- do not reference the nixpkgs channel but instead add the inputs separately

kuzzmi commented:
@SuperSandro2000 

I updated my commit with all three requested changes.  
Also, I added the font package to `top-level/all-packages.nix`, not sure tho if that is the one I had to update.
